src/Authentication/ClientCertificate/ClientCertificateClassmap.php

```diff
@@ -8,12 +8,11 @@
 
 class ClientCertificateClassmap
 {
-    public static function getCollection() : \Soap\ExtSoapEngine\Configuration\ClassMap\ClassMapCollection
+    public static function getCollection(): \Soap\ExtSoapEngine\Configuration\ClassMap\ClassMapCollection
```

Comment:
Better to fix that on the code generator side.

Reply:
There is an issue with that. But looks like nobody provided a patch yet.

Reply:
Yeah I wouldn't bother about this, we can run the phpcs fixer anyway and fix this automatically after generating the code. I'm resolving this conversation.